pipelines/LATISS/DRP.yaml

```diff
@@ -32,12 +32,12 @@ tasks:
     config:
       connections.cycleNumber: 6
 
-  updateVisitSummary:
```

Comment:
Wait, did you really mean to delete updateVisitSummary?

Reply:
No.  No, I did not.  I have no idea how that happened.